File: src/src/main/java/hackerrank/helper/System.java
package hackerrank.helper;

public final class System {

    public static InputStream in = null;
    public static PrintStream out = null;

    static public void setIn(InputStream inputStream) {
        in = inputStream;
    }

    static public void setOut(PrintStream outputStream) {
        out = outputStream;
    }
}

File: src/src/main/scala/hackerrank/helper/System.scala
package hackerrank.helper

object System {

  private var _in: InputStream = _
  private var _out: PrintStream = _

  def setIn(in: InputStream): Unit = _in = in
  def setOut(out: PrintStream): Unit = _out = out
  def in(): InputStream = _in
  def out(): PrintStream = _out
}
